### PR TITLE
Add checklist for before submitting an issue on GitHub

### PR DESCRIPTION
**Pull request by KrauseFx**
I'll update the other READMEs also, once I get ok for the lists below 👍

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 - [ ] Updated fastlane to the latest version
 - [ ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
```
Comment (ohayon) [inline .github/ISSUE_TEMPLATE.md]: Can we remove "have" from this line to match the tense of the other lines?

**Review by ohayon — DISMISSED** [review]
💯

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (KrauseFx) [inline .github/ISSUE_TEMPLATE.md]: Yep

Comment (KrauseFx) [inline .github/ISSUE_TEMPLATE.md]: Done

**Review by alexsinger — CHANGES_REQUESTED** [review]
We should update the Need Help? sections for the individual tool READMEs too.

**Comment by KrauseFx** [comment]
@alexsinger yes, see the body of the PR, I wanted to get an ok first, before copy & pasting everything

**Review by alexsinger — COMMENTED** [review]
What do you think about adding similar text from README.md to the CONTRIBUTING.md in this section: https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md#i-want-to-report-a-problem-or-ask-a-question ?

**Comment by KrauseFx** [comment]
Which part of it do you like more? The regression?

**Review by alexsinger — APPROVED** [review]
💯

**Pull request merged**